class.jetpack.php

```diff
@@ -4670,44 +4671,6 @@ public static function build_authorize_url( $redirect = false, $iframe = false )
 		return add_query_arg( $args, $api_url );
 	}
 
-	/**
-	 * Get our assumed site creation date.
-	 * Calculated based on the earlier date of either:
-	 * - Earliest admin user registration date.
-	 * - Earliest date of post of any post type.
-	 *
-	 * @since 7.2.0
-	 *
-	 * @return string Assumed site creation date and time.
-	 */
-	public static function get_assumed_site_creation_date() {
```

Comment:
Can we deprecate this method properly with a `@deprecated` docs and `_doing_it_wrong()` call, and direct to the connection manager one instead? Then after several versions, it should be safe to remove it.

Reply:
Yes good call!